src/pages/[redirect]/index.tsx

```diff
@@ -2,9 +2,10 @@ import Head from 'next/head';
 import { useRouter } from 'next/router';
 import { useEffect, useState } from 'react';
 
+import ErrorPage from '@/components/Redirect/ErrorPage';
 import LoaderTimer from '@/components/Redirect/LoaderTimer';
-import NotFound from '@/components/Redirect/NotFound';
 import RedirectFooter from '@/components/Redirect/RedirectFooter';
+import RedirectShimmer from '@/components/ShimmerEffect/RedirectShimmer';
 import { useGetOriginalUrlQuery } from '@/services/api';
```

Comment:
try puting this import above the components import statement

Reply:
Okay but you can customize I think in the package, usually I follow this order

- External Libraries 
- Internal Libraries (services, utils, custom hooks)
- Components
- Types
- Images
- CSS

Reply:
If we change the rule too many files will been change  we need to create the separate issue ticket for this issue

Reply:
Btw there are too many imports here so we keep simple like third party library, local imports and styles and images